src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -33,9 +33,12 @@
 #include <miopen/solver/problem_description_interpreter.hpp>
 #if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
 #include <ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp>
+#include <miopen/conv/heuristics/ai_heuristics.hpp>
+#include <nlohmann/json_fwd.hpp>
```

Comment:
Probably an unnecessary header

Reply:
json_fwd.hpp is indeed unnecessary, it seems to be an artifact of how we previously had KernelTuningNet integrated into MIOpen. ai_heuristics.hpp is necessary as it contains the declaration for ModelSetParams() function that is called from these two solvers.